### ISSUE TITLE
Blank page when accessing swagger

### ISSUE THREAD

**Issue by enac21** [swaggo/http-swagger#84]
Blank page when accessing localhost:port/swagger/index.html

Console log show this err:
_Uncaught DOMException: Failed to execute 'querySelector' on 'Document': '##swagger-ui' is not a valid selector._

go version: 1.18
http-swagger: 1.3.3

**Comment by Babatunde50** [comment]
Yeah, I also had this issue. Spent hours debugging, I then decided to test out the code examples they had in the docs. I was still getting same error.

**Comment by oliverasistemas** [comment]
Same error here

**Comment by ubogdan** [comment]
There is a mistake in this code 
https://github.com/swaggo/http-swagger/blob/master/example/gorilla/main.go
		`httpSwagger.DomID("#swagger-ui")` should be 		`httpSwagger.DomID("swagger-ui"),`

**Comment by hrojas2021** [comment]
> There is a mistake in this code https://github.com/swaggo/http-swagger/blob/master/example/gorilla/main.go `httpSwagger.DomID("#swagger-ui")` should be `httpSwagger.DomID("swagger-ui"),`

This did the trick

**Comment by Charlie17Li** [comment]
> https://github.com/swaggo/http-swagger/blob/master/example/gorilla/main.go

same error

**Comment by Charlie17Li** [comment]
> @ubogdan actually my "blank page" has a different error than @enac21 . The page cannot load the Swagger UI necessary resources (js, jss, images, etc). The error showing is: `Refused to apply style from 'http://localhost:1323/swagger/swagger-ui.css' because its MIME type ('text/plain') is not a supported stylesheet MIME type, and strict MIME checking is enabled. ` I will open a different issue as this is unrelated with original issue,

same error
